sous-chef/tests/unit_tests/date/test_get_due_date.py

```diff
@@ -121,7 +121,7 @@ def test_get_due_datetime_with_meal_time(
     ):
         assert frozen_due_datetime_formatter.get_due_datetime_with_meal_time(
             Weekday.monday.name, "dinner"
-        ) == create_datetime(day=24, hour=16, minute=30)
+        ) == create_datetime(day=24, hour=17, minute=15)
```

Comment:
Please, for these 2 tests, use the MealTime.dinner values for input. 

[If there are other tests like this in this file, please consider making a helper function. -> did not check if that is the case or consider how this would be done, so might be a throw-away comment]